add-on/chrome/modules/MozLoopService.jsm

```diff
@@ -29,14 +29,16 @@ const TWO_WAY_MEDIA_CONN_LENGTH = {
  * Values that we segment sharing a room URL action telemetry probes into.
  *
  * @type {{COPY_FROM_PANEL: Number, COPY_FROM_CONVERSATION: Number,
- *   EMAIL_FROM_CALLFAILED: Number, EMAIL_FROM_CONVERSATION: Number}}
+ *    EMAIL_FROM_CALLFAILED: Number, EMAIL_FROM_CONVERSATION: Number,
+ *    FACEBOOK_FROM_CONVERSATION: Number, EMAIL_FROM_PANEL: Number}}
  */
 const SHARING_ROOM_URL = {
   COPY_FROM_PANEL: 0,
   COPY_FROM_CONVERSATION: 1,
   EMAIL_FROM_CALLFAILED: 2,
   EMAIL_FROM_CONVERSATION: 3,
-  FACEBOOK_FROM_CONVERSATION: 4
+  FACEBOOK_FROM_CONVERSATION: 4,
+  EMAIL_FROM_PANEL: 5
```

Comment:
Note: This is at odds with:

https://github.com/mozilla/loop/blob/42f33dd37dc3607361dd8108fc630fd5eb71795c/add-on/panels/test/roomStore_test.js#L32

It might also be good to throw a couple of references in various places in the code to update the different locations.